### PR TITLE
Add package scoped API keys

### PR DESCRIPTION
**Pull request by ericmj**

### CODE REVIEW

File: lib/hexpm/accounts/organization.ex [hunk 1]
```diff
@@ -90,15 +90,22 @@ defmodule Hexpm.Accounts.Organization do
     {:ok, nil}
   end
 
+  def verify_permissions(%Organization{} = organization, "package", name) do
+    [organization_name, package] = String.split(name, "/", parts: 2)
+    package = Packages.get(organization_name, package)
+
+    if organization_name == organization.name && Packages.get(organization_name, package) do
```
Comment (wojtekmach) [inline lib/hexpm/accounts/organization.ex:97]: we have two calls to Packages.get/2, looks like the first one is with package name and the second one with package struct. Why is the second one necessary?

**Review by wojtekmach — COMMENTED** [review]
Awesome!

File: lib/hexpm/accounts/organization.ex (hunk 1 above)
Comment (ericmj) [reply lib/hexpm/accounts/organization.ex:97]: Good catch, this looks like a bug. I will add a test for it.

Comment (ericmj) [reply lib/hexpm/accounts/organization.ex:97]: Fixed.

**Pull request merged**